Makefile

```diff
@@ -161,7 +161,7 @@ RED=\033[0;31m
 NO_COLOUR=\033[0m
 YQ_VERSION = $(shell yq --version | $(SED) 's/^.* //g')
 
-yq_check:
+yq_check: $(eval SHELL:=/bin/bash)
```

Comment:
How does it work when your bash in `/bin/bash` is ages old ... typically on MacOS?

Reply:
In the `DEV_QUICK_START_MAC_OS.md` it specifically says:
```
#### Install latest Bash
`brew install bash`
```

So having version 4+ is already a per-requisite for building the project I believe.

In the case where bash is not stored in `/bin`

I could change it to something like:
```
SHELL=$(eval which bash)
```

Reply:
So to answer, if your bash is ages old you can't even build Strimzi, hence why the instructions say to update it.

Reply:
That is and isn't true. Yes, you need to have bash4. But there is no requirement to have bash4 in `/bin/bash`. On MacOs it is common that the system wide bash2 is `/bin/bash` but you actually use some other bash installed for example in `/usr/local/bin/bash`. But while it is common on MAcOS, it can be on other platforms as well. That is why scripts always should use `/usr/bin/env bash` in the shebang and not `/bin/bash`. So I think it is ok to expect bash4+, but we need smarter way to find it.

If `$(eval which bash)` works it would be an option (`which bash` in my case certainly points to the right bash). I guess other option might be to create some bash script `prerequisites-check.sh` for example in `tools` and put all the checks which might be needed there and just call it from the Makefile?

Reply:
I think that is a wholly reasonable suggestion. I will re-work this and create a script. I think having such a script would be useful for first time developers to the repo to ensure all their pre-reqs are installed.